images/macos/macos-12-Readme.md

```diff
@@ -117,11 +118,11 @@
 ### Browsers
 - Safari 16.0 (17614.1.25.9.10)
 - SafariDriver 16.0 (17614.1.25.9.10)
-- Google Chrome 105.0.5195.125 
+- Google Chrome 106.0.5249.61 
 - ChromeDriver 105.0.5195.52
```

Comment:
This doesn't sound good

Reply:
$ brew info chromedriver
==> chromedriver: 105.0.5195.52

Reply:
@ddobranic the major version should correlate between chrome and chromedriver

Reply:
@miketimofeev  of course, what i wanted to say is that 105.0.5195.52 version of chromedriver is the katest currently available